src/isar_robot/robotinterface.py

```diff
@@ -248,12 +221,3 @@ def _create_video(self, step: Union[TakeVideo, TakeThermalVideo]):
         video.data = data
 
         return [video]
-
-    def _update_battery_level(self) -> float:
-        self.battery_level = 100 - randrange(0, 100) * 0.5
-        return self.battery_level
-
-    def _update_pressure_level(self) -> float:
-        millibar_to_bar: float = 1 / 1000
-        self.pressure_level = (100 - randrange(0, 100) * 0.5) * millibar_to_bar
-        return self.pressure_level
```

Comment:
We should keep this code as well, we still want to generate random dummy sensor messages